packages/gestalt/src/Popover/Contents.tsx

```diff
@@ -137,19 +137,16 @@ export default function Contents({
         <div
           aria-label={accessibilityLabel}
           className={classnames(layoutStyles.relative, styles.maxDimensions, {
-            [styles.minDimensions]: width !== null,
+            [styles.minDimensions]: width != null,
```

Comment:
why this change>\?

Reply:
To remove an incosistency and TS error. `width` is read from `size` prop. `size` accepted both `underifned` and `null`. This creates confusion. If `size` shouldn't be set, then it should be `undefined`, it doesn't have to be `null`.